parlai/core/torch_agent.py

```diff
@@ -211,31 +274,41 @@ def batchify(self, obs_batch, sort=False, is_valid=lambda obs: 'text_vec' in obs
 
         cands = None
         # TODO: return candidates
-        return Batch(xs, ys, labels, valid_inds, cands)
+        return Batch(xs, x_lens, ys, y_lens, labels, valid_inds, cands)
 
-    def match_batch(self, batch_reply, valid_inds, predictions=None,
-                    candidate_preds=None):
+    def match_batch(self, batch_reply, valid_inds, output=None):
         """Match sub-batch of predictions to the original batch indices.
 
-        Batches may be only partially filled (e.g. when completing the remainder
+        Batches may be only partially filled (i.e when completing the remainder
```

Comment:
nit s/i.e/i.e./

Reply:
intentional to keep it below 79 characters :)